icu4c/source/test/intltest/units_test.cpp

```diff
@@ -351,54 +354,42 @@ void UnitsTest::testConverter() {
             continue;
         }
 
+        double maxDelta = 1e-6 * uprv_fabs(testCase.expectedValue);
```

Comment:
Changed code to calculate maxDelta's only once (I was initially thinking I might need to also handle maxDetla=Inf). Reordered lines to group them logically.